EMBRACE_Client/LibraryViewController.m

```diff
@@ -46,9 +47,24 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
     
+    ConditionSetup *conditionSetup = [[ConditionSetup alloc] init];
+    
     //Set the title to something personalized.
     if(student != nil) {
-        self.title = [[@"Hi, " stringByAppendingString:[student firstName]] stringByAppendingString:@"!"];
+        
+        //added by James for xml logging
```

Comment:
We should replace all these logging comments with comments that actually describe what is being logged.